ssz/sedes/tuple.py

```diff
@@ -0,0 +1,108 @@
+from typing import (
+    Generator,
+    Sequence,
+    Tuple,
+    TypeVar,
+)
+
+from eth_utils import (
+    to_tuple,
+)
+
+from ssz.utils import (
+    merkleize,
+    pack,
+)
+from ssz.exceptions import (
+    DeserializationError,
+    SerializationError,
+)
+from ssz.sedes.base import (
+    BaseSedes,
+    BasicSedes,
+    CompositeSedes,
+)
+
+TSerializableElement = TypeVar("TSerializable")
+TDeserializedElement = TypeVar("TDeserialized")
+
+
+class Tuple(CompositeSedes[Sequence[TSerializableElement], Tuple[TDeserializedElement, ...]]):
+
+    def __init__(self,
+                 number_of_elements: int,
+                 element_sedes: BaseSedes[TSerializableElement, TDeserializedElement]) -> None:
+
+        self.number_of_elements = number_of_elements
+        self.element_sedes = element_sedes
+
+    #
+    # Size
+    #
+    @property
+    def is_variable_length(self):
+        return self.number_of_elements > 0 and self.element_sedes.is_variable_length
+
+    def get_fixed_length(self):
+        if self.is_variable_length:
+            raise ValueError("Tuple does not have a fixed length")
+
+        return self.element_sedes.get_fixed_length()
```

Comment:
This also seems odd that it would naively return it's element's value instead of something like `self.number_of_elements * self.element_sedes.get_fixed_length()` to scale the value up to the number of elements in the tuple.

Reply:
yep, that's just plain wrong.